### ISSUE TITLE
Cannot update to the latest version of R

### ISSUE THREAD

**Issue by sburnell** [talgalili/installr#47]
I am running Windows 8.1 OS and version 3.2.0 of R and using R Studio. 

Running the following R code gets me through the whole update process. The console at the end notes that some of the MD5 checksums are off though. So, I am not updated with the 3.2.3 version. I tried update and installing several times via the console and continue to get the same message. Will try again some other time with the same R code below. 

if(!require(installr)) {
install.packages("installr"); require(installr)}

updateR()

**Comment by talgalili** [comment]
Are you running it from the Rgui? (instead of through RStudio?)
Also - the warning on MD5 doesn't say you didn't get R installed - please check again (you probably had R installed)

Tal

**Comment by talgalili** [comment]
Hi all,
This issue should be now fixed (in installr 0.17.6)
You can install it using:

```r
if (!require('devtools')) install.packages('devtools'); # make sure you have Rtools installed first! if not, then run:
#install.packages('installr')
#install.Rtools()
devtools::install_github('talgalili/installr')
```

Please let me know if this solves your issue.

Best,
Tal
